framework/core/locale/core.yml

```diff
@@ -242,6 +242,7 @@ core:
     # These translations are used for the users list on the admin dashboard.
     users:
       description: A paginated list of all users on your forum.
+      search_placeholder: Search users
```

Comment:
We organize locale alphabetically, so let's push this right above title.

Additionally, since we have another entry that translates to `Search users` (*look up `search_users_label`*) let's create a common entry under the `ref:` group and reuse that one in the other two entries, namely (`search_placeholder` and `search_users_label`).

For an example on reusing ref entries, look up `=> core.ref`